benchmark/performance_utils.py

```diff
@@ -208,6 +223,12 @@ def get_latency(self, op, *args, **kwargs):
         # average latency in ms
         return latency
 
+    def get_gbps(self, x, out, t):
+        import numpy as np
```

Comment:
We can use builtin function sum in python. No need for numpy here.

Reply:
good idea.